nilearn/__init__.py

```diff
@@ -2,15 +2,48 @@
 Machine Learning module for NeuroImaging in python
 ==================================================
 
-Nilearn aims to simplify the use of the scikit-learn in the context of
+Documentation is available in the docstrings and online at
+http://nilearn.github.io.
+
+Contents
+--------
+Nilearn aims at simplifying the use of the scikit-learn package in the context of
 neuroimaging. It provides specific input/output functions, algorithms and
 visualization tools.
 
-See http://nilearn.github.io for complete documentation.
+Submodules
+---------
+datasets                --- Utilities to download NeuroImaging datasets
+decoding                --- Decoding tools and algorithms
+decomposition           --- Includes a subject level variant of the ICA
+                            algorithm called Canonical ICA
+group_sparse_covariance --- Implementation of algorithm for sparse
+                            multi-subjects learning of Gaussian graphical models
+image                   --- Set of functions defining mathematical operations
+                            working on Niimg-like objects
+input_data              --- includes scikit-learn tranformers and tools to
+                            preprocess neuro-imaging data
+masking                 --- Utilities to compute and operate on brain masks
+mass_univariate         --- Defines a Massively Univariate Linear Model
+                            estimated with OLS and permutation test
+plotting                --- Plotting code for nilearn
+region                  --- Set of functions for extracting region-defined
+                            signals
+signal                  --- Set of preprocessing functions for time series
+
+Utilities
```

Comment:
I would remove this section alltogether, including the two lines below. In the scipy docstring case, the test they mention is scipy.test and we don't have the equivalent in nilearn since we just use nosetets.